mozilla/fx-private-relay
# Count how often the News menu is interacted with

# New feature description

This adds code to count how often different elements of the News menu are interacted with.

# How to test

Open a browser that does not send Do Not Track headers, and ensure you're running the dev server (i.e. on localhost:3000 rather than 127...) to ensure GA events are output to the console. Interact with the various elements in the News menu, and verify that events are indeed being sent.

# Checklist
